ci/do_ci.sh

```diff
@@ -291,21 +291,16 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
   # This doesn't go into CI but is available for developer convenience.
   echo "bazel with different compiletime options build with tests..."
 
-  if [[ "${TEST_TARGETS[*]}" == "//test/..." ]]; then
-    cd "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
-    TEST_TARGETS=("@envoy//test/...")
-  fi
+  cd "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
+  TEST_TARGETS=("${TEST_TARGETS[@]/#\/\//@envoy\/\/}")
```

Comment:
When running `do_ci.sh` with just a single test target for compile_time_options, does this force all the filter example tests as well?

Reply:
yes that's on purpose.

Reply:
The flow is often that a developer just wants to run a single quick test, why force them to run additional tests?

Reply:
Ah I read the first comment wrong, it runs the single test from filter example, but doesn't run additional tests.

For example, if specified test is 
`//test/common/common:base64_test`, this will run it as `@envoy//test/common/common:base64_test` from filter example repo.